commands/import_e2e_test.go

```diff
@@ -0,0 +1,309 @@
+			Id:       model.NewId(),
+			UserId:   userID,
+			Type:     model.UploadTypeImport,
+			Filename: "import1.zip",
+			FileSize: 1024 * 1024,
+		})
+		s.Require().Nil(appErr)
+		us1.Path = ""
+
+		time.Sleep(time.Millisecond)
+
+		_, appErr = s.th.App.CreateUploadSession(&model.UploadSession{
+			Id:        model.NewId(),
+			UserId:    userID,
+			ChannelId: s.th.BasicChannel.Id,
+			Type:      model.UploadTypeAttachment,
+			Filename:  "somefile",
+			FileSize:  1024 * 1024,
+		})
+		s.Require().Nil(appErr)
+
+		time.Sleep(time.Millisecond)
+
+		us3, appErr := s.th.App.CreateUploadSession(&model.UploadSession{
+			Id:       model.NewId(),
+			UserId:   userID,
+			Type:     model.UploadTypeImport,
+			Filename: "import2.zip",
+			FileSize: 1024 * 1024,
+		})
+		s.Require().Nil(appErr)
+		us3.Path = ""
+
+		err := importListIncompleteCmdF(c, cmd, nil)
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 2)
+		s.Require().Len(printer.GetErrorLines(), 0)
+		s.Require().Equal(us1, printer.GetLines()[0].(*model.UploadSession))
+		s.Require().Equal(us3, printer.GetLines()[1].(*model.UploadSession))
+	})
+}
+
+func (s *MmctlE2ETestSuite) TestImportListJobsCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.Run("no permissions", func() {
+		printer.Clean()
+
+		err := importListJobsCmdF(s.th.Client, &cobra.Command{}, nil)
+		s.Require().NotNil(err)
+		s.Require().Equal("failed to get import jobs: : You do not have the appropriate permissions., ", err.Error())
+		s.Require().Len(printer.GetLines(), 0)
+		s.Require().Len(printer.GetErrorLines(), 0)
+	})
+
+	s.RunForSystemAdminAndLocal("no import jobs", func(c client.Client) {
+		printer.Clean()
+
+		err := importListJobsCmdF(c, &cobra.Command{}, nil)
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().Len(printer.GetErrorLines(), 0)
+		s.Equal("No import jobs found", printer.GetLines()[0])
+	})
+
+	s.RunForSystemAdminAndLocal("some import jobs", func(c client.Client) {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+		limit := 2
+		cmd.Flags().Int("limit", limit, "")
+
+		_, appErr := s.th.App.CreateJob(&model.Job{
+			Type: model.JOB_TYPE_IMPORT_PROCESS,
+			Data: map[string]string{"import_file": "import1.zip"},
+		})
+		s.Require().Nil(appErr)
+
+		time.Sleep(time.Millisecond)
+
+		job2, appErr := s.th.App.CreateJob(&model.Job{
+			Type: model.JOB_TYPE_IMPORT_PROCESS,
+			Data: map[string]string{"import_file": "import2.zip"},
+		})
+		s.Require().Nil(appErr)
+
+		time.Sleep(time.Millisecond)
+
+		job3, appErr := s.th.App.CreateJob(&model.Job{
+			Type: model.JOB_TYPE_IMPORT_PROCESS,
+			Data: map[string]string{"import_file": "import3.zip"},
+		})
+		s.Require().Nil(appErr)
+
+		err := importListJobsCmdF(c, cmd, nil)
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), limit)
+		s.Require().Len(printer.GetErrorLines(), 0)
+		s.Require().Equal(job3, printer.GetLines()[0].(*model.Job))
+		s.Require().Equal(job2, printer.GetLines()[1].(*model.Job))
+	})
+}
```

Comment:
We'd need to add a happy path test for the "show one job" case, either if it has it's own command or as part of `import list jobs` if we decide on using it